package.json

```diff
@@ -35,12 +35,15 @@
   "dependencies": {
     "asar": "^0.14.0",
     "async": "^2.5.0",
```

Comment:
`async` should be removed, it's no longer needed

Reply:
Oops, I thought I did that. I'll remove it tonight.